src/mmw/js/src/modeling/models.js

```diff
@@ -270,16 +271,18 @@ var ScenarioModel = Backbone.Model.extend({
                     name: 'precipitation',
                     value: 1.0
                 }
-            ]
+            ],
+            // This is the md5 hash of the above default value
+            inputmod_hash: 'ab0d9b8a1d441b8ec7ddf81b5a49dd2d'
```

Comment:
That's a good point, and furthermore, json attributes are not guaranteed to be in any order (perhaps different browser `stringify` implementations could order them differently).  Maybe we should be very explicit about the string we hash, convert the object to an array of key/values (the attributes), order by key name, then hash?  That way we could unify across implementations/platforms/languages with higher confidence.